tests/sastoken_ut/sastoken_ut.c

```diff
@@ -1793,4 +1824,51 @@ TEST_FUNCTION(SASToken_CreateString_succeeds)
     ASSERT_ARE_EQUAL(char_ptr, umock_c_get_expected_calls(), umock_c_get_actual_calls());
 }
 
+TEST_FUNCTION(SASToken_CreateString_far_future_succeeds)
```

Comment:
"far_future" alone does not give the full idea of what we are testing here.
Could you add a comment on top: // Test added to verify code against Y2k38 issue.

Reply:
added comment and changed to long_int instead of far_future.